src/components/DappsGallery/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styles from './DappsGallery.module.scss';
+import { dappsContent } from './content/DappsGalleryContent';
+
+export default function DappsGallery() {
+  return (
+    <div className={`${styles.dappsContainer} dappsContainer`}>
+      {dappsContent.map((p, index) => (
+        <div key={index} className={`${styles.dappCard} dappCard`}>
```

Comment:
it is usually not recommended to use the index as the key: <https://legacy.reactjs.org/docs/lists-and-keys.html>

It is probably better to add a "name" property in the list and use this as the index.